samples/sudoku.cr

```diff
@@ -124,31 +124,31 @@ def sd_solve(mr, mc, s)
   ret
 end
 
-sudoku = "
-..............3.85..1.2.......5.7.....4...1...9.......5......73..2.1........4...9       near worst case for brute-force solver (wiki)
```

Comment:
These comments should stay.

Reply:
Ooh I didn't realise that these strange texts are actually comments for the specific lines. I thought they got there by accident when copying them out somewhere. I think I will add a `#` before every comment so you better notice that they are actually comments for the specific lines.